src/main.rs

```diff
@@ -1398,10 +1368,10 @@ struct BisectionResult {
 
 fn main() {
     if let Err(err) = run() {
-        match err.downcast::<ExitError>() {
-            Ok(ExitError(code)) => process::exit(code),
+        match err.downcast::<ExitStatusError>() {
```

Comment:
The `ExitError` was in main and I simply renamed it to make it clear that this is the approach to explicitly define an exit status code number.  It does not appear to be in use any longer.  Maybe we just eliminate it?